### PR TITLE
Undefined index error

### LINKED ISSUE #95: Undefined index error
If I run the language server with PHP installation that has `E_ALL` error reporting, it fails with the following error:

```
PHP Fatal error:  Uncaught Symfony\Component\Debug\Exception\ContextErrorException: Notice: Undefined index: tcp in /home/kraev/git/php-language-server/bin/php-language-server.php:22
Stack trace:
#0 /home/kraev/git/php-language-server/bin/php-language-server.php(22): Symfony\Component\Debug\ErrorHandler->handleError(8, 'Undefined index...', '/home/kraev/git...', 22, Array)
#1 {main}
  thrown in /home/kraev/git/php-language-server/bin/php-language-server.php on line 22
```

### PR DESCRIPTION
**Pull request by kaloyan-raev**
Fixes #95.

### CODE REVIEW

**Comment by felixfbecker** [comment]
You need to link issues in the commit _message_ or PR _description_ so GitHub can close them and Waffle can group them 😉

File: bin/php-language-server.php [hunk 1]
```diff
@@ -19,7 +19,7 @@
 
 cli_set_process_title('PHP Language Server');
 
-if ($options['tcp']) {
+if (isset($options['tcp'])) {
```
Comment (felixfbecker) [inline bin/php-language-server.php]: Can it be `false`?

**Comment by kaloyan-raev** [comment]
Like this in the PR description?

**Comment by felixfbecker** [comment]
Yep.

File: bin/php-language-server.php (hunk 1 above)
Comment (kaloyan-raev) [inline bin/php-language-server.php]: Will `empty` do a better job than `isset`?

Comment (felixfbecker) [inline bin/php-language-server.php]: Yes

Comment (kaloyan-raev) [inline bin/php-language-server.php]: OK. Changed it.

**Pull request merged**